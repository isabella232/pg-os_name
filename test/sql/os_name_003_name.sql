\t
SELECT v::text::os_name
FROM os_name_table;

SELECT 'invalid_name'::os_name;
SELECT ''::os_name;