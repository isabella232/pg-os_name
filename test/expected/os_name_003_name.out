\t
SELECT v::text::os_name
FROM os_name_table;
 android
 android-tv
 bada
 blackberry
 fire-tv
 ios
 linux
 macos
 roku-tv
 samsung-tv
 server
 symbian
 webos
 windows
 windows-phone
 unknown

SELECT 'invalid_name'::os_name;
ERROR:  unknown input os_name: invalid_name
LINE 1: SELECT 'invalid_name'::os_name;
               ^
SELECT ''::os_name;
ERROR:  unknown input os_name: 
LINE 1: SELECT ''::os_name;
               ^
